.github/workflows/deploy.yaml

```diff
@@ -76,8 +76,8 @@ jobs:
       - name: "build deployable API"
         run: |
           export OPENAPI_COMMAND="java -jar openapi-generator-cli.jar"
-          python regenerate_api.py
-          ls -lGh metamist
+          set -e;
```

Comment:
I was a bit surprised about this, as I thought GH Actions set `-e` etc by default — as in fact they do, per [the documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell) and [this testing](https://github.com/jmarshall/test/actions/runs/5597121893).

However, we set ` defaults: run: shell: bash -l {0}` above, which overrides this default. (I'm not sure why all this needs a login shell, but it's probably best to leave that as is!) So I'd suggest changing that default to

```
shell: bash -eo pipefail -l {0}
```

which will make for better error handling for all the job steps and mean lines 79–80 can be left as is.

Reply:
Thanks for the suggestion! I also have no idea why a login shell is used, or if it is required but I agree that we should leave that for another time.